.husky/pre-commit-phpcbf.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# Disable exit on error since we handle exit codes manually
+set +e
+
+# Get list of staged PHP files
+STAGED_PHP_FILES=$(git diff --cached --name-only --diff-filter=d | grep ".php$") || true
+
+# Exit if no PHP files are staged
+if [ -z "$STAGED_PHP_FILES" ]; then
+    exit 0
+fi
+
+echo "Running PHPCBF on staged files..."
+
+# First try to fix what we can with PHPCBF
+echo "$STAGED_PHP_FILES" | xargs vendor/bin/phpcbf
+
+echo 'does this run?'
+
+# Check the return code
+PHPCBF_STATUS=$?
+
+# If files were fixed, add them back to staging
+if [ $PHPCBF_STATUS -ne 3 ]; then
+    echo "$STAGED_PHP_FILES" | xargs git add
+    
+    if [ $PHPCBF_STATUS -eq 1 ] || [ $PHPCBF_STATUS -eq 2 ]; then
+        echo "\nShowing remaining errors after auto-fixing:"
+        echo "$STAGED_PHP_FILES" | xargs vendor/bin/phpcs
+    fi
+fi
+
+# Now run phpcs-changed to check only modified lines
+echo "Checking modified lines with phpcs-changed..."
+STAGED_FILES_SPACE_SEPARATED=$(echo "$STAGED_PHP_FILES" | tr '\n' ' ')
```

Comment:
I don't think this is needed. The expansion below should be fine with newlines rather than spaces, even with a POSIX /bin/sh.

Reply:
Seems to work without it. b2920566a68cd43832c5ee78b4335017eaeeb8cd